### ISSUE TITLE
[Bug] Game Title Text Cut in Half

### ISSUE THREAD

**Issue by Sponge-bink** [Ryujinx/Ryujinx#5570]
### Description of the issue

Title of games got cut in half with kanjis in the title

<img width="1092" alt="image" src="https://github.com/Ryujinx/Ryujinx/assets/33798984/be38c1e5-e8c3-4c71-8182-4f79db9262bc">

### Reproduction steps

Set language to Japanese, launch Ryujinx

### Log file

[Ryujinx_1.1.922_2023-08-15_10-07-37.log](https://github.com/Ryujinx/Ryujinx/files/12340741/Ryujinx_1.1.922_2023-08-15_10-07-37.log)

### OS

macOS

### Ryujinx version

1.1.988

### CPU

Apple M1 Pro

### GPU

Apple M1 Pro

### RAM

16g

**Comment by AcK77** [comment]
I'm sorry but in your screenshot the game title (in the game list) don't seems to be cutted, as " ピクミン 4 " seems to be the full text for "Pikmin 4".

**Comment by Sponge-bink** [comment]
> don't seems to be cutted [sic]

Please watch closely. The text was cut vertically, not horizontally. 

> seems to be the full text

It is the original Japanese title.

**Comment by AcK77** [comment]
> Please watch closely. The text was cut vertically, not horizontally.

Oh! Do that happen for all games ?
@IsaacMarovitz are you able to reproduce it?

**Comment by Sponge-bink** [comment]
> Do that happen for all games ?

Yes. It happens to all game whose title contains Hiragana, Katakana and Kanji.
